File: Pset1/Mario 2/mario.c
#include <stdio.h>
#include "/workspaces/142470650/Pset1/cs50.c"

int main(){
    int height = 0;
    do {
        height = get_int("Height: ");
    }while(height < 1 || height > 8);

    for(int i =0; i<height;i++){
        for(int j = 0; j<height-i-1;j++){
            printf(" ");
        }
        for(int j = 0; j<i+1; j++){
            printf("#");
        }
        printf("  ");

        for (int j = 0;j<i+1;j++){
            printf("#");
        }
        printf("\n");
    }
    return 0;
}

File: Pset6/Sentimental-Mario-More/mario.py
from cs50 import get_int

height:int = 0

while not 0<height<=8:
    height = get_int("Height: ")

for i in range(height):
    for j in range(height-i-1):
        print(' ',end='')
    for j in range(i+1):
        print('#',end='')
    print("  ",end='')
    for j in range(i+1):
        print('#',end='')
    print()


